Pino-iOS/Pino-iOS/ViewController.swift

```diff
@@ -9,11 +9,12 @@ import UIKit
 
 class ViewController: UIViewController {
 
+    var abcccc = "hi"
     override func viewDidLoad() {
         super.viewDidLoad()
         // Do any additional setup after loading the view.
+        
+        abcccc.append("454")
```

Comment:
it seems they are test code

Reply:
Ah thanks for the heads up this whole file shall be deleted